editor.planx.uk/src/ui/public/FeedbackOption.tsx

```diff
@@ -24,7 +24,6 @@ const Root = styled(Button, {
   justifyContent: "flex-start",
   ...(showArrow && {
     width: "100%",
-    maxWidth: "460px",
```

Comment:
This is the only style change made here - as this is just shy of the 600px drawer (minus padding) it looked a bit odd and out of line. 

It now shows as full width - 

<img width="1621" height="663" alt="Image" src="https://github.com/user-attachments/assets/a007d108-2faf-4bf7-91c1-4d0d84a5d01f" />